contracts/libs/gov-pool/GovPoolStaking.sol

```diff
@@ -58,6 +79,15 @@ library GovPoolStaking {
             delegatee
         );
 
+        uint256 previousDelegatorStake = micropool.latestDelegatorStake[msg.sender];
+        uint256 currentDelegatorStake = uint256(int256(stakedAmount) + stakeDifference);
+
+        uint256 rewardsDeviation = _calculateDeviation(previousDelegatorStake, stakedAmount);
```

Comment:
The deviation should not include the pending stake/unstake amounts

Reply:
closed